### PR TITLE
Add carrier slugs to marketplace-constants (bug 1087678)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1155688

### CODE REVIEW

File: dist/js/carriers.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        // Node. Does not work with strict CommonJS, but
+        // only CommonJS-like environments that support module.exports,
+        // like Node.
+        module.exports = factory();
+    } else {
+        // Browser globals (root is window).
+        root.returnExports = factory();
+    }
+}(this, function() {
+    return {"CARRIER_SLUGS": ["megafon", "kddi", "vimpelcom", "deutsche_telekom", "sprint", "o2", "tmn", "america_movil", "congstar", "telenor", "singtel", "telefonica", "etisalat", "grameenphone", "china_unicom", "mtn", "qtel", "telecom_italia_group", "smart", "hutchinson_three_group", "carrierless", "kt"], "car": {"slug": "kt", "id": 8, "name": "KT"}};
```
Comment (robhudson) [inline dist/js/carriers.js]: What's the `car` key on the end?

**Comment by robhudson** [comment]
r+wc

**Comment by chuckharmston** [comment]
Would be fantastic if we could add all the information contained within this file:

https://github.com/mozilla/zamboni/blob/master/mkt/constants/carriers.py

**Comment by chuckharmston** [comment]
See also https://bugzilla.mozilla.org/show_bug.cgi?id=1087678

File: dist/js/carriers.js (hunk 1 above)
Comment (diox) [inline dist/js/carriers.js]: lol, it's the local variable from the loop. I'll get rid of it, good catch.

**Comment by diox** [comment]
@chuckharmston my plan is to load `mpconstants.carrier` in this file, yes.

**Comment by diox** [comment]
I had not noticed your bug before, I'll dupe mine

**Comment by diox** [comment]
I'm also adding a few carriers from https://bugzilla.mozilla.org/show_bug.cgi?id=1154951 while I'm at it.

**Comment by chuckharmston** [comment]
It's nice that we have the slugs, but would also be nice to include the full names, for what it's worth.

**Comment by diox** [comment]
In the js you mean ? They're in the python lib. I didn't want to increase the size of the JS for now, so I only export the slugs because that's what the various JS projects use. We can revisit that later.

**Pull request merged**